HarpGuru/HarpFace/HarpRow/components/index.test.tsx

```diff
@@ -7,28 +7,30 @@ import { harpFaceProps } from '../../testResources'
 
 import { HarpRow } from './index'
 
-test('A blow row from a major diatonic harmonica can be rendered', () => {
-  const { getAllByText } = render(<HarpRow {...harpFaceProps} yCoord={2} />)
-
-  expect(getAllByText(DegreeIds.Root)[0]).toBeTruthy()
-  expect(getAllByText(DegreeIds.Third)[0]).toBeTruthy()
-  expect(getAllByText(DegreeIds.Fifth)[0]).toBeTruthy()
+// TODO: improve the test data setup here. We need to make the
+// test data composable from the bottom up like the types
```

Comment:
Link to this review comment in https://github.com/js-jslog/harpnative/issues/28 and then delete the TODO

Reply:
Done